### PR TITLE
fix few typo issues

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
- Change the text to "param to send to mini app".
- Fix few typo issues 

## Links
- MINI-2589

# Checklist
- [ ] I wrote/updated tests for new/changed code.
- [x] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [x] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

File: js-miniapp-sample/src/hooks/useGeoLocation.test.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import useGeoLocation from './useGeoLocation';
 
 describe('useGeoLocation', () => {
   let result;
-  const dummyCoOridinates = {
+  const dummyCoOrdinates = {
```
Comment (minh-rakuten) [inline js-miniapp-sample/src/hooks/useGeoLocation.test.js:9]: I believe `dummyCoordinates` must be correct.

**Review by minh-rakuten — APPROVED** [review]
👍 👍

File: js-miniapp-sample/src/hooks/useGeoLocation.test.js (hunk 1 above)
Comment (khairul-alam-licon) [reply js-miniapp-sample/src/hooks/useGeoLocation.test.js]: Yeah, `dummyCoordinates` looks better 👍

**Review by corycaywood — APPROVED** [review]
Looks good 👍

**Pull request merged**